android/src/main/java/com/robinpowered/Frequency/FrequencyTrackFactory.java

```diff
@@ -0,0 +1,73 @@
+package com.robinpowered.Frequency;
+
+import android.media.AudioFormat;
+import android.media.AudioManager;
+import android.media.AudioTrack;
+import android.os.Build;
+
+/**
+ * Created by Ying Hang Eng on 1/24/18.
+ */
```

Comment:
🔥 

In Java files we typically don't leave a watermark giving credit to the person who started the file. 
If anything, you might find a copyright or license comment, but not authorship.